### ISSUE TITLE
Training dataset - patch creation

### ISSUE THREAD

**Issue by anitchris** [JingyunLiang/SwinIR#50]
It would be really helpful if you could point out how to create the patches when the image size is less than 128 x 128 (the patch size mentioned in the training settings). Would we consider such images by zero padding or exclude those images as the ones present in BSD500 dataset of size 120 x 80?

**Comment by JingyunLiang** [comment]
You can pad it by reflection padding, but I remember there is all images are `481x321`. Don't downsample it when you use it for denoising.
